uber/models/__init__.py

```diff
@@ -1236,6 +1236,13 @@ def add_to_print_queue(self, attendee, printer_id, reg_station):
             if not attendee.birthdate or attendee.age_group_conf['val'] == c.AGE_UNKNOWN:
                 errors.append("Age group not recognized.")
 
+            fields = ['badge_num', 'badge_type_label', 'badge_printed_name']
+            attendee_fields = attendee.to_dict(fields)
+
+            for field in fields:
```

Comment:
We miiiight want these errors are bit more clear/English, since they'll be shown to staffers checking people in.